src/Twilio.Api/Calls.cs

```diff
@@ -183,6 +183,14 @@ public virtual Call RedirectCall(string callSid, CallOptions options)
                 request.AddParameter("StatusCallback", options.StatusCallback);
                 request.AddParameter("StatusCallbackUrl", options.StatusCallback); //workaround for issue DEVX-401
             }
+
+            if (options.RecordingStatusCallback.HasValue())
+            {
+                request.AddParameter("RecordingStatusCallback", options.RecordingStatusCallback);
+                request.AddParameter("RecordingStatusCallbackMethod", options.RecordingStatusCallbackMethod); 
```

Comment:
Is this guaranteed to have a value just because RecordingStatusCallback has one?